src/pwd/pwd.c

```diff
@@ -1,6 +1,95 @@
+#include <limits.h>
+#include <stdbool.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
 
-int main(void) {
-  printf("Hello, World!\n");
-  return 0;
-}
\ No newline at end of file
+#define NAME "pwd (canoutils)"
+#define VERSION "1.0.0"
+#define AUTHOR "Mehadav Itay (whiteapolo)"
+
+#define print_version()                                                        \
+  do {                                                                         \
+    printf("%s\nversion: %s\nby: %s\n", NAME, VERSION, AUTHOR);                \
+  } while (0)
+
+char *realpath(const char *path, char *resolved_path);
+
+int status = 0;
+bool logical = false;
+
+char buff[256];
+
+void print_pwd_pysical() {
+  getcwd(buff, 256);
+  printf("%s\n", buff);
+}
+
+void print_pwd_logical() {
+  char *env = getenv("PWD");
+  if (env == NULL) {
+    printf("pwd: cannot access PWD varibale\n");
+    status = 1;
```

Comment:
return `bool` instead of editing a global variable
```c
bool print_pwd_logical() {
  char *env = getenv("PWD");

  if (env == NULL) {
    printf("pwd: cannot access PWD varibale\n");
    return false;
  }
  printf("%s\n", env);
  return true;
}
```